cclunit-framework-source/doc/CCLUTMOCKING.md

```diff
@@ -28,11 +30,25 @@ cclutExecuteProgramWithMocks.
 &nbsp;&nbsp;&nbsp;&nbsp;The namespace under which to execute the program.
   
 Example:  
-call cclutExecuteProgramWithMocks("ccl_my_program", "\^MINE^, 1.0, ^string parameter^", "MYNAMESPACE")
+```
```

Comment:
You didn't like the javascript version? It is kind of colorful.

Reply:
Heh, I can add it to all of them if you'd prefer.  I figured since it wasn't actually JavaScript, some of the keywords might be incorrectly colored/not colored, so it might be better just to have nothing.  Some things (like strings) would benefit though.

Reply:
It stands out a lot better. 
A global search/replace for  **Example:\s*[\n\r]*```** should catch most of them. 
Do you have a markdown viewer plugin for your browser?

Reply:
I do, though I typically use the one in my IDE.  I'll update them to javascript.